.travis.yml

```diff
@@ -0,0 +1,17 @@
+language: php
+dist: trusty
+cache:
+  directories:
+    - $HOME/.composer/cache/files
+    - $HOME/.cache/composer/files
+matrix:
+  include:
+    - php: 5.6
```

Comment:
```suggestion
    - php: 5.6
    - php: 7.3
  allow_failures:
    - php: 7.3
```

Reply:
IMHO we should not let 7.3 fail currently as we need to know it's status since we should be targeting 7.3 with the new server/setup. We should add & allow `nightly` to fail.

Reply:
7.3 will fail until compatibility is finished though and we shouldn't block merging all PRs until then. Setting off on a note of it being okay to merge failing PRs is non-ideal I think.

We would then remove `allow_failures: 7.3` in the compatibility PR (and remove 5.6).

Reply:
Sounds good!